src-tauri/src/main.rs

```diff
@@ -27,8 +27,7 @@ struct Child {
 }
 
 lazy_static! {
-    static ref PATH: Mutex<String> =
-        Mutex::new("C:/Users/User/Documents/GitHub/text-editor/Folder".to_string());
+    static ref PATH: Mutex<String> = Mutex::new("%USERPROFILE%/Desktop".to_string());
```

Comment:
idk what this does but in other languages the user folders is not in english (at least on linux)